### ISSUE TITLE
Outer joins break the results table

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1027]
Run this query and make experiments an outer join:

```
<query model="genomic" view="Interaction.details.type Interaction.details.role1 Interaction.gene1.symbol Interaction.gene1.organism.name Interaction.details.role2 Interaction.gene2.primaryIdentifier Interaction.gene2.symbol Interaction.gene2.organism.name Interaction.details.experiment.name Interaction.details.experiment.publication.pubMedId" sortOrder="Interaction.gene1.symbol ASC" > 
<constraint path="Interaction.gene2.organism" op="!=" value="Interaction.gene1.organism" code="A" />
</query>
```

The experiments show up in the wrong place and the publications column is empty:

![outer](https://cloud.githubusercontent.com/assets/1663057/7931478/ee0b8e0c-0908-11e5-8d32-3975adc1fb84.png)

**Comment by julie-sullivan** [comment]
works as expected.

**Comment by joshkh** [comment]
Fixed by pull request intermine/im-tables@a55759e885d992c55d4b1e5c4180a94939979c18
